src/main/resources/application.yaml

```diff
@@ -151,6 +150,10 @@ notification:
     responseReceived:
       emailId: 90f0ed29-a616-4ce0-b4ef-108391f5d90e
       smsId: e2e166c4-3600-443d-8feb-39f2c28e8732
+  paper:
```

Comment:
Could we be clear that paper and oral cases are different notification types? So add `ORAL` to `AppealHearingType.java` and then structure the 2 different types of notifications in this yaml file so we can clearly differentiate between `paper.responseReceived.emailId` and `oral.responseReceived.emailId`

Reply:
done